pages/get-involved.html

```diff
@@ -37,7 +37,7 @@ <H3>{% trans "Start Developing Now" %}</H3>
       </li>
       <li>
         <p>
-          {% blocktrans with irc_channel_link="/irc/" %}When you're stuck, get help from our existing developers our IRC channel: #mixxx on <a href="{{ irc_channel_link }}">Freenode</a>{% endblocktrans %}
+          {% blocktrans with zulip_web_link="https://mixxx.zulipchat.com/" %}When you're stuck, get help from our existing developers our <a href="{{ zulip_web_link }}">Zulip chat</a>{% endblocktrans %}
```

Comment:
existing developers **on** our

Reply:
Fixed, thank you. I can squash commits if preferred.